tests/test_bookmarks.py

```diff
@@ -200,28 +223,28 @@ def test_run(self):
             expected_records_second_sync[stream] += updated_record
             updated_records[stream] += updated_record
 
-        # Update a Payment AFTER all other streams have been updated
-        # Payments which have already completed/cancelled can't be done so again so find first APPROVED payment
-        first_rec = dict()
-        for message in first_sync_records.get('payments').get('messages'):
-            if message.get('data')['status'] == 'APPROVED':
-                first_rec = message.get('data')
-                break
-
-        if not first_rec:
-            raise RuntimeError("Unable to find any any payment with status APPROVED")
-        first_rec_id = first_rec.get('id')
-        first_rec_version = first_rec.get('version')
-
-        updated_record = self.client.update('payments', first_rec_id, first_rec_version)
-        assert len(updated_record) > 0, "Failed to update a {} record".format('payments')
-        assert len(updated_record) == 1, "Updated too many {} records".format('payments')
+        if 'payments' in self.TESTABLE_STREAMS:
+            # Update a Payment AFTER all other streams have been updated
+            # Payments which have already completed/cancelled can't be done so again so find first APPROVED payment
+            first_rec = dict()
+            for message in first_sync_records.get('payments').get('messages'):
+                if message.get('data')['status'] == 'APPROVED':
+                    first_rec = message.get('data')
+                    break
+
+            if not first_rec:
+                raise RuntimeError("Unable to find any any payment with status APPROVED")
+            first_rec_id = first_rec.get('id')
+            first_rec_version = first_rec.get('version')
```

Comment:
We do this in another test right? Can we put a method in the base class that they all use?

Reply:
We only do updates in this test, I don't think this happens anywhere else.